src/assets/createScript.js

```diff
@@ -90,23 +90,37 @@ import { Script } from 'playcanvas';
 
 export class ${className} extends Script {
 
+    /**
+     * Called when script is about to run for the first time.
+     */
     initialize() {
-
     }
 
+    /**
+     * Called for enabled (running state) scripts on each tick.
+     * 
+     * @param {number} dt - The delta time in seconds since the last frame.
+     */
     update(dt) {
-
     }
 
-    // uncomment the swap method to enable hot-reloading for this script
-    // update the method body to copy state from the old instance
-    // swap(old) { };
+    /**
+     * Called when a Script that already exists in the registry gets redefined. If the new
+     * Script has a 'swap' method in its prototype, then it will be executed to perform
+     * hot-reload at runtime.
+     * 
+     * Uncomment the method to enable hot-reloading for this script.
+     * 
+     * @param {${className}} old - The old script instance.
+     */
+    // swap(old) {
+    // }
 }
 
-// learn more about scripting here:
-// https://developer.playcanvas.com/user-manual/scripting/
-    `.trim();
-
+/**
+ * Learn more about script {@link https://developer.playcanvas.com/user-manual/scripting/}
+ */
```

Comment:
Why would this be a JSDoc block? It's not documenting anything.

Reply:
![image](https://github.com/playcanvas/editor-api/assets/48248865/01f9df18-8721-4890-b3dc-c21050d9010d)

It was just to highlight it better. I could move it above the class to decorate it instead?

Reply:
Yeah, I think it'd work better in the class description.